### PR TITLE
Validation added

### PR DESCRIPTION
**Pull request by horaanchal**

### CODE REVIEW

File: app/src/app/add-user/add-user.component.ts [hunk 1]
```diff
@@ -43,6 +43,10 @@ export class AddUserComponent implements OnInit {
       previousExperience: this.previousExperience.nativeElement.value,
       totalExperience: this.totalExperience.nativeElement.value
     };
+    if(this.email.nativeElement.value == "" || this.firstName.nativeElement.value == "" || this.lastName.nativeElement.value == ""||this.location.nativeElement.value == ""||this.designation.nativeElement.value == ""||this.cgiCode.nativeElement.value == ""||this.previousExperience.nativeElement.value == ""||this.totalExperience.nativeElement.value == ""||this.skills.nativeElement.value == "" && this.previousExperience.nativeElement.value > this.totalExperience.nativeElement.value){
```
Comment (PurpleDragon1999) [inline app/src/app/add-user/add-user.component.ts]: Have you tested this code ? I think this.designation.nativeElement.value will not work here.

**Review by PurpleDragon1999 — COMMENTED** [review]
Have You tested these validations ?

File: app/src/app/add-user/add-user.component.ts (hunk 1 above)
Comment (horaanchal) [reply app/src/app/add-user/add-user.component.ts]: Yes I have changed it now!!

Comment (horaanchal) [reply app/src/app/add-user/add-user.component.ts]: You can merge it!

**Review by PurpleDragon1999 — APPROVED** [review]

**Pull request merged**